CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v1.1.3] 2020-01-03
```

Comment:
You are currently changing this in 1.1.2 I think

Reply:
I am fixing a bug in 1.1.2 since it is already released so I was going to create the 1.1.3 release after it as merged. I need to change the version date though :smiley:

Reply:
Unless I can delete the release but I am not sure we should do that

Reply:
No let's not delete the release - let's wait for feedback from team batman and continue then :)